src/Symfony/Component/Translation/Bridge/Phrase/Tests/PhraseProviderTest.php

```diff
@@ -869,84 +877,66 @@ public function writeProvider(): \Generator
         ];
     }
 
-    public function toStringProvider(): \Generator
+    public static function toStringProvider(): \Generator
     {
         yield 'default endpoint' => [
-            'provider' => $this->createProvider(httpClient: $this->getHttpClient()->withOptions([
-                'base_uri' => 'https://api.phrase.com/api/v2/projects/PROJECT_ID/',
```

Comment:
You kept the same `base_uri` for all cases, this doesn't affect the tests?

Reply:
No, it doesn't. The HTTP client is completely irrelevant for this particular test. But that's a good point. I'm going to replace the client with a stub.

Reply:
Done.